### PR TITLE
adding tailwind filter

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

### Added
- Filter to load Tailwind classes on all elements.

### CODE REVIEW

**Review by dadadavorin — APPROVED** [review]
Looks good to me 👌

**Review by vedranpetrac — APPROVED** [review]

File: src/Hooks/FiltersOuputMock.php
Comment (piqusy) [inline src/Hooks/FiltersOuputMock.php:1]: I'm trying to figure out why this file was named like this but to no avail.
Can you please provide some insight on that?

**Review by piqusy — APPROVED** [review]

File: src/Hooks/FiltersOuputMock.php
Comment (iruzevic) [reply src/Hooks/FiltersOuputMock.php:1]: As we have a bunch of filters all over the place,  the idea of this file is to group filters in one place and provide output data like global settings, local settings and filter settings.

Comment (piqusy) [reply src/Hooks/FiltersOuputMock.php:1]: OK, but why `...Mock` 🤔

**Pull request merged**